api_docs/writing-bots.md

```diff
@@ -212,6 +212,29 @@ None.
          content=new_content,
      ))
 ```
+
+### bot_handler.react
+
+*bot_handler.react(message, emoji_name)*
+
+will add a reaction to a message as the bot user.
+
+#### Arguments
+
+* message - a dictionary describing the message to be sent by the bot
+* emoji_name - the target emoji's human-readable name.
+
+To find an emoji's name, hover over a message to reveal three icons on the right, then click the smiley face icon. Images of available reaction emojis appear. Hover over the emoji you want, and note that emoji's text name.
+
+#### Example implementation
+
+```python
+bot_handler.react(
```

Comment:
I noticed there isn't an example for `bot_handler.send_reply`, but the [helloworld bot](https://github.com/zulip/python-zulip-api/tree/main/zulip_bots/zulip_bots/bots/helloworld) is basically an implementation of reply and react to message, so maybe we could reference/show that as an example implementation for both of these?